index.html

```diff
@@ -11,47 +11,300 @@
+                                <img src="Images/C-Vector.svg" alt="github">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="angelist"
+                            ><img src="Images/P-Vector.svg" alt="angelist">
+                        </a>
+                        </li>
+                    </ul>
+                    <button>Get my resume</button>
+                </div>
+            </div>
+            
+            <div class="skill-list">
+                <ul>
+                    <li class="frame-12">
+                        <div class="frame-7">
+                            <h4 class="languages">Languages</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-down-icon.png" alt="Arrow-Down Icon"></a>
+                        </div>
+                        <ul class="language-list">
+                            <li class="content">
+                                <img class="icon" src="Images/Ellipse-1.png" alt="JavaSript-logo">
+                                <p class="text">JavaScript</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="images/HTML-logo.png" alt="html-logo">
+                                <p class="text">HTML</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="Images/CSS-logo.png" alt="css-logo">
+                                <p class="text">CSS</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Frameworks</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Skills</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon"></a>
+                        </div>
+                    </li>
+                </ul>
+            </div>
         </section> 
-         
-    
+        <!-- contact form -->
+        <section class="contact-me">
+            
+                <h2 class="heading-2">Contact me</h2>
+                <p class="wordings">
+                  If you have an application you are interested in developing, a feature
+                  that you need built or a project that needs coding. I’d love to help
+                  with it
+                </p>
+                <form
+                  action="https://formspree.io/f/xbjwvdbquk6dt"
```

Comment:
- [ ] Make sure you are using the correct [hashid](https://help.formspree.io/hc/en-us/articles/360015130174-Getting-your-form-s-hashid-) since I am getting this message.  
![Screenshot from 2022-07-06 18-57-14](https://user-images.githubusercontent.com/87965534/177594360-2071732b-45bf-4795-87cc-8fb99e094a2b.png) 
instead of  👇 
![Screenshot from 2022-07-06 19-06-23](https://user-images.githubusercontent.com/87965534/177595073-c9027c7e-963a-4c5f-b278-1ebcf5327ae8.png)